src/commands/createSmartContractProjectCommand.ts

```diff
@@ -55,10 +55,16 @@ export async function createSmartContractProject(): Promise<void> {
     let assetType: string;
     if (smartContractLanguageItem.type === LanguageType.CONTRACT) {
         assetType = await UserInputUtil.showInputBox('Name the type of asset managed by this smart contract', 'MyAsset');
+        const regexForAssetType: RegExp = /^[A-Z]+$/i;
+        const validAssetType: boolean = regexForAssetType.test(assetType);
         if (!assetType) {
             // User has cancelled the input box
             return;
         }
+        if (!validAssetType) {
+            outputAdapter.log(LogType.ERROR, `Please ensure your asset name only consists of alphabets.`);
```

Comment:
I think this error message needs changing - maybe to something like `Invalid asset name, it should only contain lowercase and uppercase letters`

Reply:
Made the changes, thanks for the feedback!